src/core/agent/services/credentialService.ts

```diff
@@ -69,14 +69,19 @@ class CredentialService extends AgentService {
   }
 
   @OnlineOnly
-  async getCredentialDetailsById(id: string): Promise<ACDCDetails> {
+  async getCredentialDetailsById(
+    id: string
+  ): Promise<Omit<ACDCDetails, "credentialType" | "issuanceDate">> {
```

Comment:
This should be `Promise<ACDCDetails>` with this PR - this PR was to update that PR